posts/2023-03-14_hackathon_app/index.qmd

```diff
@@ -20,9 +20,6 @@ long_slug <- "2023-03-14_shiny_run_script"
 
 <!--------------- post begins here ----------------->
 
-
-
-
 We recently created a shiny application for the `admiral` hackathon in February 2023.
 The `admiral` hackathon was an event designed to make statistical programmers from the pharmaceutical industry more comfortable with the `admiral` R package which allows users to efficiently transform data from one data standard (sdtm) to another (adam).
```

Comment:
we should have these linked to cosa page for the hackathon and the admiral home page